quickstarts/microsoft.kubernetes/aks-azure-linux/azuredeploy.json

```diff
@@ -74,7 +74,10 @@
         "defaultValue": "AzureLinux",
         "allowedValues": [
           "AzureLinux",
-          "Ubuntu"
+          "AzureLinux3",

```

Comment:
You should update the metadata.json file in this folder to indicate it was updated.